data_transfer/devices/dreem.py

```diff
@@ -59,9 +63,13 @@ def download_metadata(self) -> None:
         # Note: includes metadata for ALL data records, therefore we must filter them
         all_records = dreem_api.get_restricted_list(self.session, self.user_id)
 
+        log.info(f"Total dreem records: {len(all_records)} for {self.study_site}")
+
         # Only add records that are not known in the DB based on stored filename
         # i.e. (ID and filename in dreem)
         unknown_records = [r for r in all_records if r["id"] not in set(all_hashes())]
```

Comment:
See comment at 110 below, but when using the hash, `r["id"]` becomes `r["IDEAFAST"]["hash"]`